integrations/stripe/stripe-checkout-sca.md

```diff
@@ -63,7 +63,7 @@ for DNN versions 9.0.0 and above or
 **//configuration/appSettings** the following line:
 
 ```xml
-     <add key="DnnSharpStripeApiSigningSecretKey" value="Your_Signing_Secret_Here"/>
+     <add key="DnnSharpStripeApiSigningSecretKey_YourPortalIDHere" value="Your_Signing_Secret_Here"/>
```

Comment:
You should add a new example instead of changing the existing one, maybe saying that the new one is preferred / the old one is obsolete.